### DISCUSSION TITLE
Showing off my installation

### DISCUSSION THREAD

**Discussion by rrooggiieerr** [JanM321/esphome-lg-controller#51] [Show and tell]
I had ordered my Tiny PCBs without the headers soldered on and sandwiched the ESP32 DevKitC board and the Tiny PCB together by soldering the PCB to the feet of the DevKitC. I then used heat shrink to make it a small little enclosed package:

![IMG_20240413_134718](https://github.com/JanM321/esphome-lg-controller/assets/18603534/9d6f0f85-e937-4356-a16d-e32dc24caf10)

![IMG_20240413_134757](https://github.com/JanM321/esphome-lg-controller/assets/18603534/9dfa3806-a558-4f3a-af18-26b3cf6a4892)

![IMG_20240413_134814](https://github.com/JanM321/esphome-lg-controller/assets/18603534/479ac685-08f4-4d32-b4dd-22368af58cd9)

![IMG_20240413_134856](https://github.com/JanM321/esphome-lg-controller/assets/18603534/a6959821-4238-43e2-84d5-ac1bc5ea6a76)

![IMG_20240413_134908](https://github.com/JanM321/esphome-lg-controller/assets/18603534/1bc5a599-08df-4628-88ab-46a2b597798b)

I used DuPont cables to connect the resulting contraption to my HVAC and fixed it in place with double side tape:
![IMG_20240413_140050](https://github.com/JanM321/esphome-lg-controller/assets/18603534/ebd7b598-8b9d-482f-b5f0-601a165fdee7)
![IMG_20240413_140059](https://github.com/JanM321/esphome-lg-controller/assets/18603534/4f642441-69fc-4ef5-a80b-559e2d90ffe3)

**Comment by p0358** [comment]
Nice. What is your interior unit's model? Is it classic LG Deluxe? Because if yes, then it's pretty good knowing it could just fit inside without having to worry about putting long extension cords ahead of time. Do you think it'd still manage to fit in the non-soldered scenario where the boards stacked on top of each other were taller?

**Comment by TerryFrench** [comment]
Nice, I like the large transparent heat-shrink tube as solution

**Comment by evertide** [comment]
I ordered with the headers ;-)

I also used the Home Assistant add-on, not the installation method described in the firmware section.  This by copy-pasting the template and base yaml together with as decribed in the discussions/55 with the external component below.

external_components:
  - source: github://JanM321/esphome-lg-controller
    components: ["lg_controller"]

the images:
[](url)
![airco kabel](https://github.com/user-attachments/assets/88a9d5b7-e2b2-49d6-aade-83f16fa862ea)
![airco zipbag](https://github.com/user-attachments/assets/4a03d393-7694-40be-af2a-e8f3eb28fd53)
![airco installed](https://github.com/user-attachments/assets/aceb4787-68d3-48c0-b52a-f822203e2673)

**Reply by awraeven** [reply]
@evertide which headers did you order and where? From what it looks like on your images, I have the same green connector on my  LG PC09ST NSJ unit.

**Comment by MisterP1980** [comment]
What is your interior unit's model? Is it classic LG Deluxe?
